mycroft/enclosure/gui.py

```diff
@@ -110,6 +110,14 @@ def __getitem__(self, key):
         """Implements get part of dict-like behaviour with named keys."""
         return self.__session_data[key]
 
+    def get(self, key, value=None):
+        """Implements the get method for accessing dict keys."""
+        try:
```

Comment:
Suggestion: 

Instead of this implementation just forward to __session_data.get

```python
    def get(self, *args, **kwargs):
        return self.self.__session_data.get(*args, **kwargs)
```